web/frontend/src/components/input-controller/index.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  import type { inputControllerApi } from "@viamrobotics/sdk";
+
+  export let name: string;
+  export let status: inputControllerApi.Status.AsObject;
+
+  const controlOrder = [
+    "AbsoluteX",
+    "AbsoluteY",
+    "AbsoluteRX",
+    "AbsoluteRY",
+    "AbsoluteZ",
+    "AbsoluteRZ",
+    "AbsoluteHat0X",
+    "AbsoluteHat0Y",
+    "ButtonSouth",
+    "ButtonEast",
+    "ButtonWest",
+    "ButtonNorth",
+    "ButtonLT",
+    "ButtonRT",
+    "ButtonLThumb",
+    "ButtonRThumb",
+    "ButtonSelect",
+    "ButtonStart",
+    "ButtonMenu",
+    "ButtonEStop",
+  ];
+
+  $: connected = ((eventsList: inputControllerApi.Event.AsObject[]) => {
+    for (const { event } of eventsList) {
+      if (event !== "Disconnect") {
+        return true;
+      }
+    }
+    return false;
+  })(status.eventsList);
```

Comment:
perhaps could be refactored to:

```suggestion
$: connected = status.eventsList.some(({ event }) => event !== 'Disconnect')

Reply:
Works for me! Happy to simplify and improve readability.

Reply:
Done.